Sources/nCine/Audio/Backends/SDL/SdlAudioDevice.cpp

```
@@ -0,0 +1,560 @@
+	}
+
+	void SdlAudioDevice::setGain(float gain)
+	{
+		// Applied while mixing, so the device itself always plays at unity
+		_gain = gain;
+	}
+
+	void SdlAudioDevice::updateListener(const Vector3f& position, const Vector3f& velocity)
+	{
+		static_cast<void>(velocity);
+		_listenerPos = position;
```

Cppcheck (2.21.0)

[warning] The address of variable 'X' might be accessed at non-zero index.

(objectIndex)